source/precalculus/source/03-LF/06.ptx

```diff
@@ -55,7 +55,7 @@
           <li><p> <m>6c+4a=208</m> </p></li>
           <li><p> <m>a+c=208
           </m> </p></li>
-          <li><p> <m>4a+6c=208</m> </p></li>
+          <li><p> <m>4c+6a=208</m> </p></li>
```

Comment:
Would it be good (throughout this activity) to always list the `a` term first?

Reply:
Or vice/versa (children then adults), but either way, write things consistently to reduce cognitive overload of learners?

Reply:
Consistency in order sounds good to me.

Reply:
About to make an update.